plutus-core/plutus-ir/test/PlutusIR/Transform/StrictLetRec/Tests.hs

```diff
@@ -40,7 +38,7 @@ test_letRec = runTestNestedIn path do
           (runCompilationM . runTestPass (`compileLetsPassSC` RecTerms))
           (const noProvenance <<$>> pTerm)
           "strictLetRec"
-    , pure $ testCase "traces" do
+    , embed $ testCase "traces" do
```

Comment:
`pure` doesn't work with the updated machinery, now one has to use `embed`. I liked `pure` more, but something had to pay.